src/display-console.cc

```diff
@@ -0,0 +1,68 @@
+/*
+ *
+ * Conky, a system monitor, based on torsmo
+ *
+ * Please see COPYING for details
+ *
+ * Copyright (C) 2010 Pavel Labath et al.
```

Comment:
Please change this to `Copyright (C) 2018 François Revol et al.` here and elsewhere.

Reply:
Ok I'll include it in my next round.